packages/react-native-client/README.md

```diff
@@ -0,0 +1,402 @@
+Use `<VideoRendererView />` component like this:
+
+```ts
+{
+  endpoint.videoTracks.map((track) => (
+    <VideoRendererView trackId={track.id} />
+  ));
+}
+```
+
+You can style the views to lay out them however you'd like, basic animations
+should work too.
+
+There are also some simple hooks for toggling camera, microphone and
+screensharing. Use them like this:
+
+```ts
+const { isCameraOn, toggleCamera } = useCameraState();
+const { isMicrophoneOn, toggleMicrophone } = useMicrophoneState();
+```
+
+For screencasting use `useScreencast()` hook. The local endpoint will have a new
+video track which you can render just like an ordinary video track with
+<VideoRendererView />:
+
+```ts
+const { isScreencastOn, toggleScreencast } = useScreencast();
+...
+toggleScreencast({screencastMetadata: { displayName: "Annie's desktop" }});
+```
+
+Use track metadata to differentiate between video and screencast tracks.
+
+### Android foreground service
+
+In order for the call to continue running when app is in background, you need to
+set up and start a foreground service. You can use a 3rd party library for this,
+for example [notifee](https://notifee.app/).
+
+In `AndroidManifest.xml` specify necessary permissions:
+
+```xml
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PROJECTION" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_CAMERA" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MICROPHONE" />
+```
+
+And add foreground service:
+
+```xml
+<application
+...
+>
+  ...
+  <service
+    android:name="app.notifee.core.ForegroundService"
+    android:foregroundServiceType="mediaProjection|camera|microphone" />
+</application>
+```
+
+Then to start the foreground service:
+
+```ts
+import notifee, { AndroidImportance } from '@notifee/react-native';
+
+const startForegroundService = async () => {
+  if (Platform.OS === 'android') {
+    const channelId = await notifee.createChannel({
+      id: 'video_call',
+      name: 'Video call',
+      lights: false,
+      vibration: false,
+      importance: AndroidImportance.DEFAULT,
+    });
+
+    await notifee.displayNotification({
+      title: 'Your video call is ongoing',
+      body: 'Tap to return to the call.',
+      android: {
+        channelId,
+        asForegroundService: true,
+        ongoing: true,
+        pressAction: {
+          id: 'default',
+        },
+      },
+    });
+  }
+};
+```
+
+Don't forget to also stop the service when the call has ended:
+
+```ts
+notifee.stopForegroundService();
+```
+
+Also add this code in your `index.js` to register the service:
+
+```js
+notifee.registerForegroundService((notification) => {
+  return new Promise(() => {});
+});
+```
+
+## Credits
+
+This project has been built and is maintained thanks to the support from
+[dscout](https://dscout.com/) and [Software Mansion](https://swmansion.com).
```

Comment:
Let's remove it from here, and only leave copyright as for example here: https://github.com/fishjam-cloud/web-client-sdk/tree/main/packages/react-client#copyright-and-license